packages/nextjs/components/burnerwallet/QrCodeReader.tsx

```diff
@@ -54,6 +55,9 @@ const QrCodeReader = () => {
             </div>
           ) : (
             <div className="max-w-[90%] w-[300px] h-[300px] fixed top-0 left-0 right-0 bottom-0 m-auto z-[100]">
+              <button className="pointer-events-auto" onClick={() => setIsQrReaderOpen(false)}>
```

Comment:
Nitpick: to me feels more natural the X to close at the right side of the camera square. Maybe just personal preference!